src/boot/boot.janet

```diff
@@ -3283,9 +3283,10 @@
   (var *debug* false)
   (var *compile-only* false)
 
-  (if-let [jp (getenv-alias "JANET_PATH")] (setdyn :syspath jp))
```

Comment:
May I ask why you changed if-let to when-let? I consider it an antipattern to use when with only one form. Also, I wouldn't say I like changes outside of the PR main domain.

Reply:
I had chosen to do it that way for newly added form, as a means of communicating I don't think the else branch should be considered. I can change it back if that's considered bad Janet style.

Reply:
I actually learned this from @bakpakin's code so that I would say his call. Anyway, I do not find it proper to change things to look the same as the new thing you have added, at least without the note about it.

Reply:
Doesn't matter IMO, I would accept the change.

Reply:
I was thinking about this a lot, even go through the boot.janet to check `when` vs `if` for one branch stuff and it seems it is not so clear as I saw it originally. So sorry about a bikeshed, definitely should not be the main argument.

Still, I think, you should not change the code, around your change to match your style.